manim/mobject/types/vectorized_mobject.py

```diff
@@ -2056,7 +2059,7 @@ def construct(self):
 
     """
 
-    def __init__(self, *vmobjects, **kwargs):
+    def __init__(self, *vmobjects: Any, **kwargs: Any) -> None:
```

Comment:
Any? Shouldn't it also be `*vmobjects: VMobject`?

Reply:
It works with `VMobject` according to mypy. Implemented.